src/org/zaproxy/zap/extension/bugTracker/BugTrackerGithubIssue.java

```diff
@@ -0,0 +1,148 @@
+                body.append(" *DESCRIPTION* \n" + alerts[i].getDescription().toString() + "\n\n");
+            }
+            if(alerts[i].getOtherInfo().length() > 0 ) {
+                body.append(" *OTHER INFO* \n" + alerts[i].getOtherInfo().toString() + "\n\n");
+            }
+            if(alerts[i].getSolution().length() > 0 ) {
+                body.append(" *SOLUTION* \n" + alerts[i].getSolution().toString() + "\n\n");
+            }
+            if(alerts[i].getReference().length() > 0 ) {
+                body.append(" *REFERENCE* \n" + alerts[i].getReference().toString() + "\n\n");
+            }
+            if(alerts[i].getParam().length() > 0 ) {
+                body.append(" *PARAMETER* \n" + alerts[i].getParam().toString() + "\n\n");
+            }
+            if(alerts[i].getAttack().length() > 0 ) {
+                body.append(" *ATTACK* \n" + alerts[i].getAttack().toString() + "\n\n");
+            }
+            if(alerts[i].getEvidence().length() > 0 ) {
+                body.append(" *EVIDENCE* \n" + alerts[i].getEvidence().toString() + "\n\n\n\n");
+            }
+
+            if(alerts[i].getRisk() >= 0 ) {
+                labels.append("Risk: " + alerts[i].MSG_RISK[alerts[i].getRisk()] + ", ");
+            }
+            if(alerts[i].getConfidence() >= 0 ) {
+                labels.append("Conf: " + alerts[i].MSG_CONFIDENCE[alerts[i].getConfidence()] + ", ");
+            }
+            if(alerts[i].getCweId() >= 0 ) {
+                labels.append("CWE: " + alerts[i].getCweId() + ", ");
+            }
+            if(alerts[i].getWascId() >= 0 ) {
+                labels.append("WASC: " + alerts[i].getWascId() + ", ");
+            }
+        }
+        title.replace(title.length()-2, title.length()-1, "");
+        dialog.addTextField(index, FIELD_TITLE, title.toString());
+        dialog.addMultilineField(index, FIELD_BODY, body.toString());
+        dialog.addTextField(index, FIELD_LABELS, labels.toString());
+        dialog.addTextField(index, FIELD_ASSIGNEE, "");
+        dialog.addTextField(index, FIELD_USERNAME, "");
+        dialog.addTextField(index, FIELD_PASSWORD, "");
+    }
+
+    public void raiseOnTracker(String repo, String title, String body, String labels, String assignee, String username, String password) throws IOException {
+        GitHub github = GitHub.connectUsingPassword(username, password);
+        try {
+            GHRepository repository = github.getRepository(repo);
+            GHIssueBuilder issue = repository.createIssue(title);
+            issue.body(body);
+            issue.assignee(assignee);
+            String[] labelArray = labels.split(",\\s");
+            for( int i = 0; i < labelArray.length; i++ ) {
+                issue.label(labelArray[i]);
+            }
+            String response = issue.create().toString();
+            if(response.contains("Unauthorized")) {
+                System.out.println("Check your credentials.");
```

Comment:
These error messages should be shown in the UI (most users wont see them otherwise).
You can use View.showWarningDialog(..)
The messages should also be internationalised